opentsdb/getmetrics_opentsdb.py

```diff
@@ -170,22 +255,22 @@ def chunks(l, n):
         yield l[i:i + n]
 
 
-def setloggerConfig(logLevel):
+def set_logger_config(level):
     """Set up logging according to the defined log level"""
     # Get the root logger
-    logger = logging.getLogger(__name__)
+    logger_obj = logging.getLogger(__name__)
     # Have to set the root logger level, it defaults to logging.WARNING
-    logger.setLevel(logLevel)
+    logger_obj.setLevel(level)
     # route INFO and DEBUG logging to stdout from stderr
```

Comment:
We can add a log formatter to show timestam of the log and log level. Some other agent should have that code. The user has asked for it before